.github/workflows/nodejs.yml

```diff
@@ -0,0 +1,26 @@
+name: Node CI
+
+on: [push]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        node-version: [8.x, 10.x, 12.x]
```

Comment:
Should be OK with just the latest node version?

Reply:
hmmmm so I'm a little hesitant to take it out since it was included by default from the github starter node workflow, but i agree i think we can just test it on 12.x because that is the current node version